### PR TITLE
Iss 27

### PR DESCRIPTION
**Pull request by HomoCodens**
Adds parameter `relative_bar_chart` to `tsplot` to create normalized bar charts.

See #27

### CODE REVIEW

**Comment by HomoCodens** [comment]
We should probably make it a theme parameter.

**Comment by HomoCodens** [comment]
Actually nevermind, `group_bar_chart` is also a `tsplot` parameter.

File: R/tsplot.R [hunk 1]
```diff
@@ -178,6 +186,16 @@ tsplot.list <- function(...,
...
   
   if(is.null(theme)) theme <- init_tsplot_theme()
+  
+  if(relative_bar_chart) {
+    # Normalize ts
+    if(group_bar_chart) {
+      m <- max(unlist(tsl))
+      tsl <- lapply(tsl, '/', m)
+    } else {
+
```
Comment (mbannert) [inline R/tsplot.R]: what's this empty else good far??

**Comment by mbannert** [comment]
ok finally got it. tricked myself here :) . learning: relative is not a very good way to display bar when negative contributions are possible

**Pull request merged**